### ISSUE TITLE
Invalid color string for 'color_*'…

### ISSUE THREAD

**Issue by r3lgar** [endaaman/tym#25]
`tym` from `git/master`.

STDERR:

```
 % tym
** Message: 17:05:37.136: Invalid color string for 'color_0': #131517
** Message: 17:05:37.136: Invalid color string for 'color_1': #b14c4e
** Message: 17:05:37.136: Invalid color string for 'color_2': #13a24c
** Message: 17:05:37.136: Invalid color string for 'color_3': #b19336
** Message: 17:05:37.136: Invalid color string for 'color_4': #3868bd
** Message: 17:05:37.136: Invalid color string for 'color_5': #8c1db4
** Message: 17:05:37.136: Invalid color string for 'color_6': #3293b4
** Message: 17:05:37.136: Invalid color string for 'color_7': #b9bbbd
** Message: 17:05:37.136: Invalid color string for 'color_8': #000000
** Message: 17:05:37.136: Invalid color string for 'color_9': #f25555
** Message: 17:05:37.136: Invalid color string for 'color_10': #00d952
** Message: 17:05:37.136: Invalid color string for 'color_11': #f2c130
** Message: 17:05:37.136: Invalid color string for 'color_12': #3880ff
** Message: 17:05:37.136: Invalid color string for 'color_13': #b90cf2
** Message: 17:05:37.136: Invalid color string for 'color_14': #30c1f2
** Message: 17:05:37.136: Invalid color string for 'color_15': #ffffff
```

theme.lua:

```
local bg = "#373c41"

return {
	color_foreground = "#b9bbbd",
	color_bold = "#ffffff",
	color_cursor = "#ffffff",
	color_cursor_foreground = "#373c41",
	color_highlight = "#37465e",
	color_highlight_foreground = "#b9bbbd",
	color_background = bg,
	color_0 = "#131517",
	color_8 = "#000000",
	color_1 = "#b14c4e",
	color_9 = "#f25555",
	color_2 = "#13a24c",
	color_10 = "#00d952",
	color_3 = "#b19336",
	color_11 = "#f2c130",
	color_4 = "#3868bd",
	color_12 = "#3880ff",
	color_5 = "#8c1db4",
	color_13 = "#b90cf2",
	color_6 = "#3293b4",
	color_14 = "#30c1f2",
	color_7 = "#b9bbbd",
	color_15 = "#ffffff",
}
```

What's wrong?

**Comment by endaaman** [comment]
Sorry it is for my easy mistake. It has been fixed at 9eee1fa82d5b222faef7ec81bb92b342879e9563

Please upgrade to latest master.

**Comment by r3lgar** [comment]
>Please upgrade to latest master.

Fixed. Thanks.
